modules/ppcp-onboarding/resources/js/settings.js

```diff
@@ -1,11 +1,18 @@
 document.addEventListener(
```

Comment:
`settings.js` is complex and difficult to read (and not about onboarding anymore) but I am not sure if there is a simple way to improve that.

Reply:
yes, it became complex, but I also don't know how to improve it, maybe separate PR is needed. But let's skip for now.